src/create-context-store.js

```diff
@@ -9,18 +16,32 @@ import {
 } from './hooks';
 import createStore from './create-store';
 
-export default function createContextStore(model, config) {
+export default function createContextStore(model, config = {}) {
   const StoreContext = createContext();
 
-  function Provider({ children, initialData }) {
+  function Provider({ children, initialData, ...injections }) {
```

Comment:
I have given this a bit more thought and wonder if we should instead allow you to provide a configuration object instead of just injections. If a configuration object was defined in the `createContextStore` call then the prop via the `Provider` would override it. 

This would provide with a great deal of flexibility to have build/runtime defined configurations.

Consumers would also be able to do their own merge strategies on the `Provider` prop config too should they like.

i.e.

```typescript
<MyStore.Provider config={{ injections: { foo: 'bar' } }}>
```

Reply:
being able to pass the configuration runtime, could be useful, although some of the options dont make much sense to be defined, for example "disableImmer"

when you create the contextStore or localStore you define your actions based on the fact that Immer will be on or off, so It doesn't make sense to change It's value runtime

devTools doesn't seem to make sense either, because it could be set conditionally on the store creation based on env variables, having toggles runtime for it, doesn't seem that useful

name of the store could make sense if we we're reusing the same "context/local store" on many levels or in parallel ( same level on the tree ) and we want to distinguish them on dev tools... That one really seems to add some value

perhaps instead of passing a [StoreConfig](https://easy-peasy.now.sh/docs/api/store-config.html), perhaps just a sub-set of it?

Reply:
Yep, good point. Happy to document that the API will only support a subset of `StoreConfig`. 👍